dags/iop/parser.py

```diff
@@ -93,3 +99,9 @@ def _get_extracted_arxiv_eprint_value(self, article):
             self.logger.error("The arXiv value is not valid.", dois=self.dois)
         except AttributeError:
             self.logger.error("No arXiv eprints found", dois=self.dois)
+
+    def _parse_page_to_int(self, value: str):
+        try:
+            return [int(value)]
```

Comment:
why do you always return lists even if it's a single value of another type? Is it something required by the next processing steps?

Reply:
Also maybe it would be worth moving this function to utils (or another common module you have) cause you might use it in other places (in inspire we have the same function called `maybe_int`)